constraints.txt

```diff
@@ -99,7 +99,7 @@ imagesize==1.2.0
 importlib-metadata==1.1.3
 iniconfig==1.0.1
 intake-xarray==0.3.1
-intake==0.6.0
+intake==0.6.2
```

Comment:
gets ride of a warning that happens every time we import `intake`:

```
/home/OliWM/miniconda3/envs/fv3net/lib/python3.7/site-packages/intake/__init__.py:39: UserWarning: deprecation: intake.Catalog now references the base class intake.catalog.base.Catalog
 If you want to open a generic URL, you should use intake.open_catalog
  warnings.warn('deprecation: intake.Catalog now references the base class intake.catalog.base.Catalog\n '
```

Reply:
Building with this new constraints file works for the fv3net image but not the prognostic run image, can you try put this intake requirement in `docker/prognostic_run/requirements/base.txt` and then regenerate the constraints with `pip-compile`? Maybe it will be more consistent that way.

Reply:
I reverted this change. Requirements changes should be isolated to separate PRs.